examples/controlplane/single-node/controlplane.yaml

```diff
@@ -60,21 +59,24 @@ spec:
   ntp:
     servers: []
   users:
-  - name: ubuntu
+  - name: capo
+    sudo: "ALL=(ALL) NOPASSWD:ALL"
     sshAuthorizedKeys:
     - "$MACHINE_CONTROLLER_SSH_PUBLIC_FILE_CONTENT"
+  # For more information about these values,
+  # refer to the Kubeadm Bootstrap Provider documentation.
   initConfiguration:
     localAPIEndpoint:
       advertiseAddress: '{{ ds.ec2_metadata.local_ipv4 }}'
       bindPort: 6443
     nodeRegistration:
       name: '{{ local_hostname }}'
+      criSocket: "/var/run/containerd/containerd.sock"
```

Comment:
You shouldn't need this here (unless you are attempting to support k8s v1.13), since kubeadm should autodetect the criSocket for you by default.